dspace/config/emails/share_submission

```diff
@@ -0,0 +1,26 @@
+## E-mail with a download link
+##
+## Parameters: {0} is expanded to submission link
+##             {1} helpdesk email
+##             {2} helpdesk phone number
+##             {3} DSpace name short
+##             {4} DSpace name
+##             {5} DSpace UI url
+##
+## See org.dspace.core.Email for information on the format of this file.
+##
+#set($subject = 'Submission share link')
+To pass your submission to another user give them the following link:
+ ${params[0]}
+
+If you have trouble please contact
+${params[1]} or call us at ${params[2]}
+
+
+${params[3]} Team
+
+_____________________________________
```

Comment:
can't we use something like
${config.get('dspace.name')} ?

Reply:
Fixed, thank you.